test/integration/variable_test.rb

```diff
@@ -79,4 +79,10 @@ def test_hash_with_default_proc
   def test_multiline_variable
     assert_equal 'worked', Template.parse("{{\ntest\n}}").render!('test' => 'worked')
   end
+
+  def test_global_options_on_variable
+    template = Template.parse(%({{test}}), global_filters: ['truncate: 5'])
```

Comment:
"end-user" is a vague term.  Do you mean the template author?  Because there is no where in the template for them to specify a global filter.

If you mean the programmer that is embedding liquid into their application, then no, they aren't necessarily more familiar with liquid syntax, and allowing them to use any ruby code gives them more control.  E.g. they can easily call `CGI.escapeHTML(input)` if they want to use this for escaping, but they couldn't easily add taint checking.